### DISCUSSION TITLE
New Source: Fusebill

### DISCUSSION THREAD

**Discussion by dttg** [airbytehq/airbyte#33142] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
I need to be able to receive and send data to Fusebill
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 
No specific old version, ok using lastest.

## Describe the context around this new connector
* Which team in your company wants this integration, what for? This helps us understand the use case.
Our Revops team needs to send and collect data from Fusebill. 
* How often do you want to run syncs?
Daily is ok, hourly would be perfect!
* If this is an API source connector, which entities/endpoints do you need supported?

## Describe the alternative you are considering or using
Evaluating Rudder or Airbyte. 
What are you considering doing if you don’t have this integration through Airbyte?
Building it ourselves.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200368060087111/1200401507761834) by [Unito](https://www.unito.io)

**Comment by marcosmarxm** [comment]
https://developer.fusebill.com/reference

**Comment by marcosmarxm** [comment]
@dttg we recently released our CDK to improve build new sources, do you want to build this connector? I can help you setting up the env and give some tips to implement it (btw I'm Brazilian too! any question you can call me here or on our Slack channel). In any case, I added the connector to our roadmap for future implementation.

**Comment by cheyura** [comment]
**Webhook-based? (no/partially/yes)**
Partially.https://support.fusebill.com/hc/en-us/articles/115004831668-Configure-Webhooks

**Available authentication modes (API key/Oauth/other)**
Authenticate your account when using the API by including your secret API key in the request. You can manage your API key in Fusebill's UI. Your API key carries many privileges. Do not share API keys in publicly accessible areas such GitHub or client-side code.

Creating an API Key
You must create and use an API Key in order to access the Fusebill API.

In the Fusebill UI, navigate to Settings-> Integrations -> API Key.
Click Create Key to generate an API key.
The generated key is shown in boldface below. You will need to copy and paste this key into your authorization header inside your application.
Creating an account

Has Sandbox environment ?
Before you do anything, you should work with your implementation specialist to get a sandbox account. One will be used during the initial implementation process, so a member of your team should already have access to it.


How to populate the account with data?
API, Sandbox API,UI

Available streams for sync
CUSTOMERS, SUBSCRIPTIONS, PURCHASES, TRACKING, PAYMENTS, INVOICE AND STATEMENTS, CATALOG


Integration supports incremental sync?
No

Other information/blockers
May need some help from fusebill support for sandbox

**Comment by cheyura** [comment]
@sherifnada  Ready for implementation

[a comment by an automated account is not shown]
